packages/react/examples/codesandbox/components/NoticeChoice/index.html

```diff
@@ -1,22 +1,30 @@
 <!--
-  Copyright IBM Corp. 2022
-
-  This source code is licensed under the Apache-2.0 license found in the
-  LICENSE file in the root directory of this source tree.
- -->
+@license
+Copyright IBM Corp. 2020
+This source code is licensed under the Apache-2.0 license found in the
+LICENSE file in the root directory of this source tree.
+-->
 
 <!DOCTYPE html>
-<html>
-<head>
-	<title>@carbon/ibmdotcom-react example</title>
-	<meta charset="UTF-8" />
-  <link rel="stylesheet" href="https://1.www.s81c.com/common/carbon-for-ibm-dotcom/tag/v1/latest/plex.css"/>
-  <link rel="stylesheet" href="https://1.www.s81c.com/common/carbon-for-ibm-dotcom/tag/v1/latest/grid.css"/>
-</head>
-
-<body>
-  <div id="app"></div>
-
-	<script src="./src/index.js"></script>
-</body>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <title>carbon-web-components example with React</title>
```

Comment:
Maybe keep the title as `@carbon/ibmdotcom-react example`?
```suggestion
    <title>@carbon/ibmdotcom-react example</title>
```